pkgs/data/machine-learning/resnet/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenvNoCC, fetchurl }:
+let
+  srcs = {
+    resnet50 = fetchurl {
```

Comment:
Maybe, switch to `fetchzip` so that we can reuse a single store path from different derivations. This should be better than unpacking it very time.

Reply:
followed lead from https://github.com/NixOS/nixpkgs/blob/076860e0340a5e4a909b9a710e186508b14d1c90/pkgs/data/machine-learning/mnist/default.nix, which was convenient for `tensorpack` as their [code unzips directly into memory](https://github.com/tensorpack/tensorpack/blob/031e698d3fa0c9afc60d981c3dbd0b1502e92e0c/tensorpack/dataflow/dataset/mnist.py#L31)

Reply:
This is not what you do:
https://github.com/NixOS/nixpkgs/pull/61253/commits/e04a6a6363b8fe10d4f3f5eb6f6fc21986864067#diff-42048d91301b80ddde645ac463ec646dR137-R138
But, sure, it's your choice.

Reply:
I think we'll leave as-is for now, but re-evaluate if/when other packages use this to see if any need tar.gz or if they all need to call `tar`.